src/lib/Containers/Inquiry.tsx

```diff
@@ -82,6 +83,17 @@ const ResponseRateLine = styled.View`
   margin-top: 5;
 `
 
+// tslint:disable-next-line:no-empty-interface
+interface Props extends RelayProps {}
```

Comment:
I assume this is a conscious choice to not just throw `RelayProps` below?

Reply:
I think it's mainly consistency, but then an empty interface is also odd. I can remove it.